enrollment-server/src/main/java/com/wultra/app/enrollmentserver/controller/api/admin/AdminController.java

```diff
@@ -45,25 +49,40 @@ public class AdminController {
 
     private final OperationTemplateService operationTemplateService;
 
+    private final ObjectMapper objectMapper;
+
     @GetMapping("/template")
     public ObjectResponse<List<TemplateDetailResponse>> templates() {
```

Comment:
Not an ideal format of response - you cannot add any attributes to the response object. I recommend creating one more wrapper object and keeping the list under the `templates` key in it.

Reply:
I guess you mean (as used in others APIs)
API should return `ObjectResponse<TemplateListResponse>`
and create an object
`TemplateListResponse extends ArrayList<TemplateDetailResponse> `

Reply:
Ok, I see.